daprdocs/content/en/operations/configuration/install-certificates.md

```diff
@@ -84,12 +84,12 @@ spec:
 When using Windows containers, the sidecar container is started with admin privileges, which is required to install the certificates. This does not apply to Linux containers.
 {{% /alert %}}
 
+{{% /codetab %}}
```

Comment:
Just noticed this: alert shortcode (lines 83-85) doesn't work within codetab shortcode, so we need to change to:

**Note**: When using Windows containers, the sidecar container is started with admin privileges, which is required to install the certificates. This does not apply to Linux containers.

Reply:
https://v1-8.docs.dapr.io/operations/configuration/install-certificates/ is is working here, right? @hhunter-ms

Reply:
hm sort of? it puts the note into a code block kind of window. I can update later.

Reply:
Ah, I see what you mean. OK, thank you!

Reply:
Hey @hhunter-ms, this is fixed in my recent PR https://github.com/dapr/docs/pull/2628